src/js/background/assignManager.js

```diff
@@ -1,4 +1,5 @@
 const assignManager = {
+  MENU_REOPEN_IN: "reopen-in-container",
```

Comment:
When this lands we should file a follow up bug to have a JS file to control context menus rather than have it here.